### DISCUSSION TITLE
Projects (beta) Cross organization tracking

### DISCUSSION THREAD

**Discussion by fjetter** [community/community#6116] [Projects and Issues]
Is it possible to link repositories and track issues of a public organization?

**Comment by Tyriar** [comment]
I'd like to track issues in the xtermjs org in a microsoft org project. For my personal use case I'd be happy with an option to explicitly add an outside organization so auto complete doesn't need to searching all of github, or even just not supporting auto complete outside the org but allowing adding via `org/repo#num`

**Reply by rkoster** [reply]
This is partially possible via the web UI: https://github.com/github/feedback/discussions/6212#discussioncomment-1775149
